corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -150,7 +152,8 @@ def build_case_id_query_data(self):
         if self.module.is_multi_select():
             data.ref = "."
             data.nodeset = self._get_multi_select_nodeset()
-            data.exclude = self._get_multi_select_exclude()
+            if not case_search_sync_cases_on_form_entry_enabled_for_domain(self.domain):
```

Comment:
Hey Simon is this still applicable? Not quite sure what you're referring to

Reply:
I'm suggesting that instead of calling `case_search_sync_cases_on_form_entry_enabled_for_domain` you use the value of `self.exclude_relevant` to determine whether to add in `data.exclude` or not.

I think that is the correct thing to do since it is the same condition, only applied to multi-select case lists.